controllers/util/util.go

```diff
@@ -24,6 +25,10 @@ var (
 
 	EraserImage = flag.String("eraser-image", "ghcr.io/azure/eraser:latest", "eraser image")
 	EraserArgs  = utils.MultiFlag([]string{})
+
+	eraserPullPolicy    = flag.String("eraser-pull-policy", string(corev1.PullIfNotPresent), "pull policy for eraser image")
```

Comment:
i thought we wanted to move these parts out of this PR?

Reply:
@sozercan I misunderstood. Prior to this PR, the Pull Policy is hard coded so there is no option for the user (or tests) to change it. I thought you meant "leave the flags for now, then implement the configmap later and remove the flags". We need either the flags or the configmap. Should I put this PR on hold until the configmap is implemented?